change/react-native-windows-2020-04-24-14-23-47-fix-textinput-brush-update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

Comment:
This needs to be set to "patch" in order to trigger a publish in 61. This is super not apparent right now, and we are working to fix this.  Let me see if I have permissions to fix this for you quickly. Thanks for spending the time to backport to this branch.

Reply:
Understood. Removing AutoMerge in the meantime.